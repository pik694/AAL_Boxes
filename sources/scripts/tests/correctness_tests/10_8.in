10
13 73 11
15 85 64
68 3 7
39 87 27
47 74 32
84 8 17
57 38 68
22 91 44
18 90 50
28 88 38

