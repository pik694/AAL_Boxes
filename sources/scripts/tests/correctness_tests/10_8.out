2 0 8 5 1 7 3 9 4 6 
717806->694515
